### DISCUSSION TITLE
Can I use it with android?

### DISCUSSION THREAD

**Discussion by roeilulavy** [ScarletsFiction/SFMediaStream#26] [Q&A]
Hello!
I want to use this SFMediaStream with my android app I'm working on..
Is this possible? 
@StefansArya

**Comment by StefansArya** [comment]
Hi!
It's possible to create Android app with this library using Cordova/Phonegap, I usually use [Capacitor](https://capacitorjs.com/docs/android) to make Android apps. or just PWA.

I recommend to target Android >= 6.0 as the [WebView](https://developer.chrome.com/docs/multidevice/webview/) (Android System WebView) usually get auto-updated.
Android lower than 5.0 (Lollipop, API 21) should be blacklisted as the WebAudio API may be incompatible with WebView lower than [v37](https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API#browser_compatibility).

But If you're targetting your app for user below Android 6.0 (and >=5.0), I think it's better to remind the user to update the Chrome browser and Android System WebView just in case if it wasn't updated.

I also think the current (most updated) Chrome browser v102 is now not compatible for Android 5.0.1. So.. to avoid security issues and some incompatibilities from the browser-side, it's better to target Android >=6.0 (Marshmallow, API 23) anyway.
